packages/booter-lb3app/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@loopback/booter-lb3app",
+  "version": "1.0.0",
```

Comment:
Let's use `1.0.0-1` as the starting version so that `1.0.0` can be released as part of `loopback-next`.